src/hooks/useAllLiquidtyMiningCampaings.ts

```diff
@@ -0,0 +1,281 @@
+import { gql, useQuery } from '@apollo/client'
+
+import { Pair, Token, TokenAmount } from '@swapr/sdk'
+import { useMemo } from 'react'
+import { useActiveWeb3React } from '.'
+import { SubgraphLiquidityMiningCampaign, SubgraphSingleSidedStakingCampaign } from '../apollo'
+import { useNativeCurrency } from './useNativeCurrency'
+
+import { toLiquidityMiningCampaign, toSingleSidedStakeCampaign } from '../utils/liquidityMining'
+import { DateTime, Duration } from 'luxon'
+import { useAllTokensFromActiveListsOnCurrentChain } from '../state/lists/hooks'
+import { getAddress, parseUnits } from 'ethers/lib/utils'
+import { useKpiTokens } from './useKpiTokens'
+import { PairsFilterType } from '../components/Pool/ListFilter'
+
+const SINGLE_SIDED_CAMPAINGS = gql`
+  query($userId: ID) {
+    singleSidedStakingCampaigns {
+      id
+      owner
+      startsAt
+      endsAt
+      duration
+      locked
+      stakeToken {
+        id
+        symbol
+        name
+        decimals
+        totalSupply
+        derivedNativeCurrency
+      }
+      rewards {
+        token {
+          address: id
+          name
+          symbol
+          decimals
+          derivedNativeCurrency
+        }
+        amount
+      }
+      singleSidedStakingPositions(where: { stakedAmount_gt: 0, user: $userId }) {
+        id
+      }
+      stakedAmount
+      stakingCap
+    }
+  }
+`
+const REGULAR_CAMPAIGN = gql`
+  query($userId: ID) {
+    liquidityMiningCampaigns {
+      address: id
+      duration
+      startsAt
+      endsAt
+      locked
+      stakingCap
+      rewards {
+        token {
+          derivedNativeCurrency
+          address: id
+          name
+          symbol
+          decimals
+        }
+        amount
+      }
+      stakablePair {
+        id
+        reserveNativeCurrency
+        reserveUSD
+        totalSupply
+        reserve0
+        reserve1
+        token0 {
+          address: id
+          name
+          symbol
+          decimals
+        }
+        token1 {
+          address: id
+          name
+          symbol
+          decimals
+        }
+      }
+
+      stakedAmount
+      liquidityMiningPositions(where: { stakedAmount_gt: 0, user: $userId }) {
+        id
+      }
+    }
+  }
+`
+
+export function useAllLiquidtyMiningCampaings(
```

Comment:
'Campai**gn**s' instead of 'Campai**ng**s'. To be changed in every name within the file.